tests/functional/ext/dict_init_mutate.py

```diff
@@ -36,3 +36,6 @@ def update_dict(di):
     di["one"] = 1
 
 update_dict(config)
+
+config = {}
+globals()["config"]["dir"] = "bin"
```

Comment:
Should we also test `globals()["config"]`?

Reply:
I'm sure there a thousand variations that could be added. In this case this was just the simplest piece of technically valid code I could think of to trigger the missing line.